protocols/S3/stationA_protocol3_lysates_S3.ot2.apiv2.py

```diff
@@ -98,23 +96,41 @@
     }
 
 # Function definitions
-def run_info(start, end, parameters = dict()):
+def run_info(start, end):
     info = {}
-    hostname = subprocess.run(
-        ['hostname'],
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE
-    ).stdout.decode('utf-8')
 
-    info["RobotID"] = hostname
+    par = {
+        "NUM_SAMPLES" : NUM_SAMPLES,
+        "LYSATE_LABWARE" : LYSATE_LABWARE,
+        "PLATE_LABWARE" : PLATE_LABWARE,
+        "VOLUME_LYSATE" : VOLUME_LYSATE,
+        "BEADS" : BEADS,
+        "LANGUAGE" : LANGUAGE,
+        "RESET_TIPCOUNT" : RESET_TIPCOUNT
+    }
+    info["RobotID"] = os.uname()[1]
```

Comment:
why is it better uname that hostname function? I think it's easier to know which hostname does that know that second tuple of uname is the machine name..?